content/en/tracing/_index.md

```diff
@@ -26,46 +26,25 @@ disable_toc: true
 
 {{< wistia 2kgmb9wbsr >}}
 
-## What is APM?
+## What is Datadog APM?
 
- Datadog Application Performance Monitoring (APM or tracing) provides you with deep insight into your application's performance - from automatically generated dashboards for monitoring key metrics, like request volume and latency, to detailed traces of individual requests - side by side with your logs and infrastructure monitoring. This documentation covers Agent v6 only, for more information on setting up APM tracing with Agent v5, [see the APM with Agent v5 doc][1].
+ Datadog Application Performance Monitoring (APM or tracing) provides you with deep insight into your application's performance - from automatically generated dashboards for monitoring key metrics, like request volume and latency, to detailed traces of individual requests - side by side with your logs and infrastructure monitoring.
 
- Tracing allows you to see a single request moving across your system and infrastructure, and it gives you systematic data about precisely what is happening to this request. This gives you better insight into your systems interdependencies, it allows you to see where requests are taking the longest, it gives you visual cues for accurately troubleshooting outages and other system issues, and it allows you to see detailed traces of individual requests-side by side with your logs and infrastructure monitoring.
+ When a request is made to an application, Datadog can see the traces across a distributed system, and we can show you systematic data about precisely what is happening to this request. This gives you better insight into your systems interdependencies, it allows you to see where requests are taking the longest, it gives you visual cues for accurately troubleshooting outages and other system issues, and it allows you to see detailed traces of individual requests - side by side with your logs and infrastructure monitoring.
```

Comment:
We end both of these with `side by side with your logs and infrastructure monitoring` and the phrase `detailed traces of individual requests`. We can scrap this entire second paragraph it's the same info. Only language I'd take here is updating `to detailed traces of individual requests across a distributed system`.